README.rst

```diff
@@ -10,13 +10,11 @@ uncertainties
    :target: https://pepy.tech/project/uncertainties
 .. image:: https://codecov.io/gh/lebigot/uncertainties/branch/master/graph/badge.svg
    :target: https://codecov.io/gh/lebigot/uncertainties/
-.. image:: https://travis-ci.com/lebigot/uncertainties.svg?branch=master
-   :target: https://travis-ci.com/lebigot/uncertainties
-.. image:: https://ci.appveyor.com/api/projects/status/j5238244myqx0a0r?svg=true
-   :target: https://ci.appveyor.com/project/lebigot/uncertainties
+.. image:: https://img.shields.io/github/actions/workflow/status/lmfit/uncertainties/python-package.yml?logo=github%20actions
+   :target: https://github.com/lmfit/uncertainties/blob/main/.github/workflows/python-package.yml
```

Comment:
this badge in the readme results in a "4040 file not found", so that doesn't seem correct

Reply:
Aah, good catch. I think the branch name should be `master` instead of `main`. I'll work on this.